src/components/media/MediaModal.jsx

```diff
@@ -1,30 +1,27 @@
-import { yupResolver } from "@hookform/resolvers/yup"
 import PropTypes from "prop-types"
 import { useState } from "react"
 import { useForm, FormProvider } from "react-hook-form"
 import { useRouteMatch } from "react-router-dom"
-import * as Yup from "yup"
 
 import { MediaAltText } from "components/media/MediaAltText"
 import MediasSelectModal from "components/media/MediasSelectModal"
 import { MediaCreationModal } from "components/MediaCreationModal/MediaCreationModal"
 
 import { getMediaDirectoryName } from "utils/media"
 
-const MediaModal = ({ onClose, onProceed, type, showAltTextModal = false }) => {
+const MediaModal = ({
+  onClose,
+  onProceed,
+  type,
+  showAltTextModal = false,
+  onExternalProceed,
```

Comment:
from what i can tell, in both the markdown and tiptap editor, `onExternalProceed` appears to be doing the exact same thing as `onProceed` - this seems to suggest that this parameter is superfluous and should be removed

Reply:
Yep, removed in 516be476